### ISSUE TITLE
[$80][MSFT-103] [Submit] Tab button is unable to navigate in Submission Upload Page.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2807]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges/30095032/submit
2. Hit the Tab button. 
3. Keep o hitting until it selects the Submission Upload page.
4. Then hit space bar it will open the new pop up window. 
5.  Then Hit the Tab button again to navigate the icons listed, using up down arrow key.

**Expected Result**
Tab button should highlight/select the Icons present in the pop up window of Submission upload page.

**Actual Result**
Tab button is being trapped, and not working as expected.

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: 
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=ae14bf9a%2D590b%2D4bb4%2Da40a%2D4c6822c880a4&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FCreeya%2FBug%20103

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096175 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096175 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by crazyk07** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097855 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097855 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097855 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by romitgit** [comment]
Not able to register for this challenge.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097855 has been updated - it has been assigned to Timor.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by romitgit** [comment]
@crazyk07 I am not able to hit submit. It's disabled. 

And when I directly open the url /submit, it says you are not authorized.

**Comment by veshu** [comment]
@romitgit please use local challenge id 30055275 with TonyJ user with password appirio123

**Comment by romitgit** [comment]
ok thanks!

**Comment by sushilshinde** [comment]
@romitgit please submit asap we are closing in

**Comment by romitgit** [comment]
ok

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by sushilshinde** [comment]
@romitgit can you submit PR or un-assign yourself?

**Comment by rashmi73** [comment]
@sushilshinde for file submission we use filestack-js library, I guess it is not our control because UI is initiated by 
".picker" method.

I may be wrong.

**Comment by sushilshinde** [comment]
@rashmi73 you are right

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097855 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
@crazyk07 @sushilshinde I have solved it finally. Phew! 😅 Please review.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097855 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by veshu** [comment]
@r0hit-gupta I can navigate with tab behind the submission popup as well. When popup is opened the tab should be navigated behind the popup.

**Comment by r0hit-gupta** [comment]
@veshu fixed the issue. Please review.

**Comment by drasticdpk** [comment]
@r0hit-gupta @veshu, Tab is not focusing on  **Select file to Upload** control, can we make it focusable  so user can able to select the file. 
Here is the reference link 
https://monosnap.com/file/ngYt4McVR80ZtXzSPNVl7Eo2VoLut2

**Comment by r0hit-gupta** [comment]
@drasticdpk thank you for the video reference. @veshu I have made some changes. Please review. Thanks!

**Comment by tosha5252** [comment]
tab is still behaving the same way as in the video in dev env…


tested on win10/chrome

**Comment by r0hit-gupta** [comment]
@tosha5252 did you try this locally?

**Comment by tosha5252** [comment]
@r0hit-gupta  no...it's not working in the dev env

**Comment by r0hit-gupta** [comment]
@tosha5252 try clearing your cache or opening in incognito mode. It is working fine for me in dev env.

[Video.zip](https://github.com/topcoder-platform/community-app/files/3520473/Video.zip)

**Comment by r0hit-gupta** [comment]
@tosha5252 also working for me on the beta URL https://beta-community-app.topcoder.com/challenges/30099617/submit

**Comment by drasticdpk** [comment]
@tosha5252 @r0hit-gupta, yes it is working as expected on test/beta environment.

**Comment by tosha5252** [comment]
Verified working correctly

Win10/Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097855<br/><br/>```This is an automated message for crazyk via Topcoder X```
